beaker-puppet.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |s|
   # Documentation dependencies
   s.add_development_dependency 'yard'
   s.add_development_dependency 'markdown'
+  s.add_development_dependency 'activesupport', '< 6.0.0'
```

Comment:
I don't think we need the `markdown` gem at all, can we just drop it (and activesupport)?

```
$ git --no-pager grep markdown
beaker-puppet.gemspec:  s.add_development_dependency 'markdown'
```

Reply:
👍 for removing if is not used